scripts/Mansion.gd

```diff
@@ -1106,29 +1107,21 @@ func _on_end_pressed():
 	#####          Outside Events
 
 
-	for guild in globals.guildslaves:
-		var slaves = globals.guildslaves[guild]
-		count = round(clamp(0.25, 0.75, 0.01 * slaves.size() + rand_range(0.1, 0.4)) * slaves.size())
-		###---Added by Expansion---### Ank BugFix v4a
-		var idx
-		var removeId
-		var tempslave
-		for i in range(count):
-			idx = randi() % (slaves.size()*7/4) % slaves.size()
-			removeId = slaves[idx].id
-			for id in slaves[idx].relations:
-				tempslave = globals.state.findslave(id)
-				if tempslave:
-					tempslave.relations.erase(removeId)
-			slaves.remove(idx)
-		###---End Expansion---###
-			 
-		if slaves.size() < 4:
-			get_node("outside").newslaveinguild(2, guild)
-		if slaves.size() < 10 && randf() < 0.85:
-			get_node("outside").newslaveinguild(1, guild)
-		if randf() < 0.5:
-			get_node("outside").newslaveinguild(1, guild)
+	for i in globals.guildslaves:
```

Comment:
This section appears to be a significant regression to the guild slave mechanics. I do not see anything of value added.

Reply:
Reverted back